db/migrate/20170424020035_change_events_datetime_col_to_date.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeEventsDatetimeColToDate < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :events, :datetime, :date
```

Comment:
@marianmosley I think you just meant to remove the column "time"?
And actually I think the column should probably be named 'start_date' for clarity.

Also, I think 'duration' should probably be an integer (minutes), if that makes sense to you.